src/components/scenes/GettingStartedScene.tsx

```diff
@@ -565,10 +575,18 @@ const Footnote = styled(EdgeText)(theme => ({
   includeFontPadding: false
 }))
 
-const ButtonFadeContainer = styled(View)(theme => ({
-  position: 'absolute',
-  bottom: theme.rem(3.25),
-  left: 0,
-  right: 0,
-  zIndex: 1
-}))
+const ButtonFadeContainer = styled(View)<PlatformProps>(theme => props => {
+  const { platform } = props
+  // HACK: Address iOS/Android parity mismatches when the animation fires
+  return platform === 'android'
```

Comment:
Why do this as a prop? Why not `const platform = Platform.OS` right inside the component? Then the parent doesn't have to care about the hacks.